kroxylicious-test-tools/src/test/java/io/kroxylicious/test/tester/DefaultKroxyliciousTesterTest.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.test.tester;
+
+import org.apache.kafka.clients.admin.Admin;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Well that turned out to be bigger than expected see 5d58785be432b6023d5f3d57815235055748e4f9 (arguably should be a separate PR but since you raised it here and its a mechanical change I've done it here)